content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,250 @@ the product page, you can see the following sections:
 
-*   Creates a `ServiceRole` `service-viewer` which allows read access to any service in the `default` namespace that has
-the `app` label
-set to one of the values `productpage`, `details`, `reviews`, or `ratings`. Note that there is a
-constraint specifying that
-the services must have one of the listed `app` labels.
+Run the following command to create a mesh-level access control policy `deny-all` policy
+in the root namespace, A policy in the root namespace will select all workloads in the mesh.
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_service_viewer.yaml" >}}
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`. If you have changed it to a different value from the
+default `istio-system`, Please update the value accordingly in the following examples.
 
-*   Creates a `ServiceRoleBinding` that assigns the `service-viewer` role to all services in the `istio-system` and `default` namespaces.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_bind_service_viewer.yaml" >}}
-
-You can expect to see output similar to the following:
-
-{{< text plain >}}
-servicerole "service-viewer" created
-servicerolebinding "bind-service-viewer" created
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: istio-system
+spec:
+  {}
+EOF
 {{< /text >}}
 
-Now if you point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`). You should see the "Bookinfo Sample" page,
-with the "Book Details" section in the lower left part and the "Book Reviews" section in the lower right part.
+The `{}` in the policy means `deny-all`, you can also set other mesh-level policy for your own needs.
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+Now you should see `"RBAC: access denied"`. The error shows that the configured `deny-all`
+policy is working as intended, and Istio doesn't have any rules that allow any access to
+workloads in the mesh.
 
-### Cleanup namespace-level access control
+### Clean up mesh-level access control
 
-Remove the following configuration before you proceed to the next task:
+Disable the configured access control with the following command before continuing:
 
 {{< text bash >}}
-$ kubectl delete -f @samples/bookinfo/platform/kube/rbac/namespace-policy.yaml@
+$ kubectl delete authorizationpolicy.security.istio.io/deny-all -n istio-system
 {{< /text >}}
 
-## Enforcing Service-level access control
+## Enforce namespace-level access control
```

Comment:
Enforce => Enforcing

Reply:
@rcaballeromx is asking me to change this from "enforcing" to "enforce", can we reach agreements before making any further changes?

Reply:
@rcaballeromx can decide. I was looking at the current docs. But there is inconsistency here and there. Mostly it has "ing" after a verb in the title, but there are exceptions. For example: https://preliminary.istio.io/docs/tasks/security/health-check/#deploying-citadel-with-health-checking, https://preliminary.istio.io/docs/tasks/traffic-management/circuit-breaking/